bazel/repository_locations.bzl

```diff
@@ -761,8 +761,8 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.grpc_credentials.aws_iam",
             "envoy.tracers.opencensus",
         ],
-        release_date = "2020-12-09",
-        cpe = "cpe:2.3:a:haxx:curl:*",
+        release_date = "2021-03-31",
```

Comment:
It appears you need the 30th here.

Reply:
It's odd, that this was 00:08 GMT on the 31st - it seems really unwise that bazel build is using local time for package validation.

Reply:
I'm very close to either writing or finding a browser extension that will convert Github times to UTC, but for now I inspect the release date and get the `datetime` value from the source

![image](https://user-images.githubusercontent.com/246317/114471673-dc212180-9ba5-11eb-8746-a50dacf6b9ff.png)

Reply:
The date value expected by bazel-envoy build is 2021-03-30, if you inspect the failure log. Which only makes sense if CI times are PDT or later.